web-gateway/app/controllers/SearchController.java

```diff
@@ -29,34 +31,38 @@
     private final SearchService searchService;
 
     private final Boolean showInlineInstruction;
+    private HttpExecutionContext ec;
 
     @Inject
-    public SearchController(Configuration config,
+    public SearchController(Config config,
```

Comment:
Yes, and they did.

`Configuration` may still be used but is deprecated. In order to reduced the number of deprecation warnings I made the update to the suggested new API.